types/ember-fastboot/global.d.ts

```diff
@@ -0,0 +1,12 @@
+import url from 'url';
+
+interface ModuleRegistry {
```

Comment:
This bit was kind of weird and hard to figure out, but it allowed me to get around not having types for the `FastBoot` global yet.